python/stretch_diagnostics_tests/power_tests/test_POWER_battery_health.py

```diff
@@ -0,0 +1,180 @@
+                        baseline_reading['current'].append(p.status['current'])
+                        if p.status['voltage'] < 12.2:
+                            print('Battery was not fully charged')
+                            break
+                        if len(baseline_reading['voltage']) == sample:
+                            timeout = 0
+                            print('Baseline reading done')
+                            baseline_voltage = numpy.mean(baseline_reading['voltage'])
+                            baseline_current = numpy.mean(baseline_reading['current'])
+                            time.sleep(0.1)
+                            self.start_stress()
+
+                    # Only collect votlage readings when current draw is above 3.9A but less than 5A
+                    if 3.9 < p.status['current'] < 5 and len(baseline_reading['voltage']) >= sample:
+                        load_reading['voltage'].append(p.status['voltage'])
+                        load_reading['current'].append(p.status['current'])
+
+                        if len(load_reading["voltage"]) == sample:
+                            load_voltage = numpy.mean(load_reading['voltage'])
+                            load_current = numpy.mean(load_reading['current'])
+                            self.kill_stress()
+
+                            voltage_drop = float(round((baseline_voltage - load_voltage), 3))
+                            esr = float(
+                                round(((baseline_voltage - load_voltage) / (load_current - baseline_current)), 3))
+                            battery_sts = ('Voltage Drop: %f ESR: %f' % (voltage_drop, esr))
+
+                            self.test.log_data('baseline_voltage', float(baseline_voltage))
+                            self.test.log_data('baseline_current', float(baseline_current))
+
+                            self.test.log_data('load_voltage', float(load_voltage))
+                            self.test.log_data('load_current', float(load_current))
+
+                            self.test.log_data("voltage_drop", voltage_drop)
+                            self.test.log_data("esr", esr)
+
+                            print('\n########################################################')
+
+                            if voltage_drop <= 0.5:
+                                health = ("Batteries are in Great Health")
+                            elif 0.5 < voltage_drop <= 0.75:
+                                health = ('Batteries are in ok health..they are not as good as brand new\n'
+                                          'but, they should meet expected run times.'
+                                          'You can run a repair cycle to improve battery perfomance')
+                            else:
+                                health = ('Batteries are not in good health!!\n'
+                                          'Recommended to run repair cycle on batteries')
```

Comment:
Could you make all the assertion messages into a single-line statement without the new line character for the diagnostics UI to print reports cleanly?